hc/api/tests/test_it_sends_telegram.py

```diff
@@ -0,0 +1,38 @@
+from hc.test import BaseTestCase
+from datetime import timedelta as td
+import time
+from hc.api.models import Channel, Check, Notification
+from hc.api.transports import Sms, Telegram
+from django.core import mail
+from django.utils import timezone
+
+class SmsTelegramTestCase(BaseTestCase):
+
+    def _setup_data(self, kind, value, status="down", email_verified=True):
```

Comment:
- Use the `setUp()` method instead. It's best for this purpose. You can then unset the fields in `tearDown()`